### PR TITLE
Add Inventaire.io importer

### PR DESCRIPTION
**Pull request by hellekin**
Here is a go at creating an importer for inventaire.io.

### CODE REVIEW

**Comment by hellekin** [comment]
Hi @mouse-reeve, I went through the failing jobs and cleaned the files accordingly. No functional change was made except for the list of mandatory fields that I restored to the actual fields used during import from CSV exported from Inventaire.io and used in the bespoke importer. 

Tests are now failing as follows:

```shell

% docker-compose run --rm web python manage.py test bookwyrm.tests.importers.test_inventaire_import
Starting bookwyrm_redis_activity_1 ... done
Starting bookwyrm_redis_broker_1   ... done
Starting bookwyrm_db_1             ... done
Starting bookwyrm_celery_worker_1  ... done
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
..FFF...
======================================================================
FAIL: test_handle_import_twice (bookwyrm.tests.importers.test_inventaire_import.InventaireImport)
re-importing books
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/bookwyrm/tests/importers/test_inventaire_import.py", line 176, in test_handle_import_twice
    self.assertEqual(shelf.books.first(), self.book)
AssertionError: None != <<class 'bookwyrm.models.book.Edition'> key=None title='Example Edition'>

======================================================================
FAIL: test_handle_imported_book (bookwyrm.tests.importers.test_inventaire_import.InventaireImport)
Inventaire.io import added a book, this adds related connections
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/bookwyrm/tests/importers/test_inventaire_import.py", line 122, in test_handle_imported_book
    self.assertEqual(shelf.books.first(), self.book)
AssertionError: None != <<class 'bookwyrm.models.book.Edition'> key=None title='Example Edition'>

======================================================================
FAIL: test_handle_imported_book_already_shelved (bookwyrm.tests.importers.test_inventaire_import.InventaireImport)
Inventaire.io import added a book, this adds related connections
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/bookwyrm/tests/importers/test_inventaire_import.py", line 149, in test_handle_imported_book_already_shelved
    self.assertIsNone(self.user.shelf_set.get(identifier="read").books.first())
AssertionError: <<class 'bookwyrm.models.book.Edition'> key=None title='Example Edition'> is not None

----------------------------------------------------------------------
Ran 8 tests in 21.225s

FAILED (failures=3)
Destroying test database for alias 'default'...
```

**Comment by hellekin** [comment]
(Note that tests were already failing like this before the change. I think it's because the tests are wrong: I'm not familiar with Python or Django.)

File: bookwyrm/tests/importers/test_inventaire_import.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        self.assertEqual(retry_items[0].index, 0)
+        self.assertEqual(import_items[0].data["Book Id"], "wd:Q831517")
+        self.assertEqual(retry_items[1].index, 1)
+        self.assertEqual(retry_items[1].data["Book Id"], "wd:Q4351947")
+
+    @responses.activate
+    def test_import_data(self):
+        """resolve entry"""
+        import_job = self.importer.create_job(self.user, self.csv, False, "unlisted")
+        book = models.Edition.objects.create(title="Test Book")
```
Comment (mouse-reeve) [inline bookwyrm/tests/importers/test_inventaire_import.py:86]: another `with patch("bookwyrm.preview_images.generate_edition_preview_image_task.delay"):`

**Review by mouse-reeve — CHANGES_REQUESTED** [review]
I think these lines will get your tests running, I'll try running it myself later today as well

**Comment by mouse-reeve** [comment]
I'm going through and closing pull requests that are stalled or out of date -- if you'd like to get this PR back in action, I'm super happy to re-open and work on getting it in sync with the current state of main ❤️

**Pull request closed without merging**